velox/functions/sparksql/tests/XxHash64Test.cpp

```diff
@@ -26,6 +26,11 @@ class XxHash64Test : public SparkFunctionBaseTest {
   std::optional<int64_t> xxhash64(std::optional<T> arg) {
     return evaluateOnce<int64_t>("xxhash64(c0)", arg);
   }
+
+  template <typename T, typename Seed>
+  std::optional<int64_t> xxhash64WithSeed(Seed seed, std::optional<T> arg) {
```

Comment:
This function is only called once, hence, doesn't need to be a template. Did you plan to add more calls with different template parameters?